Standards/scs-XXXX-w1-Availability-Zones-Standard.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "SCS Availability Zone Standard: Implementation and Testing Notes"
+type: Supplement
+track: IaaS
+status: Draft
+supplements:
+  - scs-XXXX-vN-Availability-Zones-Standard.md
+---
+
+## Automated Tests
+
+The standard will not exclude small deployments and edge deployments, that both will not meet the requirement for being divided into multiple Availability Zones.
```

Comment:
I'm still of the opinion that 'preclude' is a better choice of word than 'exclude', see
https://www.merriam-webster.com/dictionary/preclude
https://www.merriam-webster.com/dictionary/exclude

In particular:

> Preclude is often used in legal writing, where it usually refers to making something legally impossible. A new law may be passed by Congress to preclude any suits of a certain kind against a federal agency, for example. Some judges have found that the warnings on cigarette packs preclude any suits against the tobacco companies by lung-cancer sufferers. But there are plenty of nonlegal uses as well. Bad weather often precludes trips to the beach, and a lack of cash might preclude any beach vacation at all.

I don't think that 'exclude' would be used in these instances.